### ISSUE TITLE
Require AT testing in Design Principles and Requirements 

### ISSUE THREAD

**Issue by paulairy** [w3c/silver#344]
Are there additional Design Principles and Requirements that should be included in the WCAG 3 project?

- Regarding testing on assistive technology, are there specific applications you require the tests to conducted on?

**Comment by lauracarlson** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by alastc** [comment]
Proposed response: 
<hr>
In terms of requirements, if a good way of testing a guideline is with assistive technology, that can be specified. However, it is not necessary to specify this in the requirements document. It is not always the best way to test something.

**Comment by alastc** [comment]
Response above agreed.
